doc/SAI-Proposal-SAI-PTF.md

```diff
@@ -0,0 +1,132 @@
+# SAI PTF
+
+| Title       | SAI PTF         |
+| ----------- | --------------- |
+| Authors     | Intel           |
+| Status      | In review       |
+| Type        | Standards track |
+| Created     | 07/10/2021      |
+| SAI-Version | 1.9             |
+
+## Overview
+
+SAI PTF (Packet Testing Framework) is an autogenerated Python based dataplane testing framework. It is using thrift features to call C-based SAI functions in Python scripts. The main goal of the framework is to reduce the time spent on writing wrapper functions for SAI which could be generated based on SAI headers.
+
+Apart from the framework there's a set of tests separated in per-feature files that are using SAI PTF to verify correctness of SAI layer implemented on top of any switch.
```

Comment:
Update this section highlighting what is being contributed.
1. Auto-generation framework using existing SAI meta infra
2. New PTF tests using this new framework

From here you can add details in an Autogeneration section and PTF tests section

Reply:
Importantly, we cannot say verify correctness of SAI layer implementation. Some of the vendors may pushback.
Just say we are trying to open-source a large number of tests to verify any SAI implementation in a standard manner.